client/src/annoMatrix/crossfilter.ts

```diff
@@ -50,151 +70,126 @@ export default class AnnoMatrixObsCrossfilter {
 
   See API documentation in annoMatrix.js.
   **/
-  // @ts-expect-error ts-migrate(7006) FIXME: Parameter 'colSchema' implicitly has an 'any' type... Remove this comment to see the full error message
-  // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types --- FIXME: disabled temporarily on migrate to TS.
-  addObsColumn(colSchema, Ctor, value) {
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any --- FIXME: disabled temporarily on migrate to TS.
-    const annoMatrix = (this as any).annoMatrix.addObsColumn(
-      colSchema,
-      Ctor,
-      value
-    );
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any --- FIXME: disabled temporarily on migrate to TS.
-    const obsCrossfilter = (this as any).obsCrossfilter.setData(
-      annoMatrix._cache.obs
-    );
+  addObsColumn<T extends DataframeValueArray>(
+    colSchema: AnnotationColumnSchema,
+    Ctor: new (n: number) => T,
```

Comment:
Is it possible to use `TypedArrayConstructor` here? From `client/src/common/types/arraytypes.ts:21`

CC: @bkmartinjr

Reply:
you can, but this is also fine - it is the classic TS style...